dotnet/src/SemanticKernel.Core/Functions/MethodLevelFunction.cs

```diff
@@ -33,7 +33,7 @@ namespace Microsoft.SemanticKernel;
 /// with additional methods required by the kernel.
 /// </summary>
 [DebuggerDisplay("{DebuggerDisplay,nq}")]
-internal sealed class NativeFunction : ISKFunction, IDisposable
+internal sealed class MethodLevelFunction : ISKFunction, IDisposable
```

Comment:
I would like to understand the reasons for this change, if possible. `Native Function` is domain naming which we currently use, and as far as I know, we don't have any plans at the moment to change it. This kind of renaming should be done in separate PR, powered by additional documentation (ADR). But in all cases, I'm not sure it's on our radar at the moment.

The issue, which is attached to this PR, is about methods `SKFunction.FromNativeMethod` and `SKFunction.FromNativeFunction`, which could be renamed to `FromMethod`.